### ISSUE TITLE
Makefile: Add rule to export doc

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#851]
We need to give our users a way to consult the documentation before running Nyxt.
Otherwise, if Nyxt fails to start or if there is any other accessibility issue, they won't be able to access it.

**Comment by jmercouris** [comment]
We'd have to make the documentation as part of a different system that Nyxt itself then depends on. We'd also have to limit our code to cl-markup only... This is a bit of a problem.

**Comment by jmercouris** [comment]
I believe 99% of our issues are related to GTK, so if we remove the dependency for GTK, by just loading the base system, we should be able to export the documentation, most of the time, no?

**Comment by Ambrevar** [comment]
I was suggesting adding a rule to the Makefile, not changing the .asd.
See how we extract the version number.  We can do the same with the documentation.

**Comment by jmercouris** [comment]
You will still need to load some Lisp system to parse the manual files, that means loading CL-Markup and whatever other embedded Lisp we have within the source.

**Comment by Ambrevar** [comment]
It's not necessary, for instance the tutorial content can be retrieved
by calling (tutorial-content).
I'll send a patch, that will make it clearer.

**Comment by Ambrevar** [comment]
Done in de3080c78e5225a60434ffdf99030b4cfd5f629b.
I had a fix a few functions for them to work without a *browser*.
With that in place, doc generation is trivial.
